pkg/storaged/stratis-details.jsx

```diff
@@ -703,6 +894,33 @@ export function start_pool(client, uuid, show_devs) {
                         unlock_with_keydesc(key_desc);
                 }));
     }
+
+    if (stopped_props.key_description &&
+        stopped_props.key_description.t == "(bv)" &&
+        stopped_props.key_description.v[0] &&
+        stopped_props.key_description.v[1].t == "(bs)" &&
+        stopped_props.key_description.v[1].v[0]) {
+        key_desc = stopped_props.key_description.v[1].v[1];
+    }
+
+    if (stopped_props.clevis_info &&
+        stopped_props.clevis_info.t == "(bv)" &&
+        stopped_props.clevis_info.v[0] &&
+        stopped_props.clevis_info.v[1].t == "(b(ss))" &&
+        stopped_props.clevis_info.v[1].v[0]) {
+        has_clevis = true;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)